### PR TITLE
timeout buffer become configurable

### PR DESCRIPTION
**Pull request by saartochner**

### CODE REVIEW

File: src/lumigo_tracer/utils.py [hunk 1]
```diff
@@ -96,6 +100,9 @@ def config(
         step_function or os.environ.get("LUMIGO_STEP_FUNCTION", "").lower() == "true"
     )
     Configuration.timeout_timer = timeout_timer
+    Configuration.timeout_timer_buffer = timeout_timer_buffer or float(
```
Comment (efimk-lu) [inline src/lumigo_tracer/utils.py:103]: What happens if someone will write there a string or space by accident?

Comment (saartochner) [reply src/lumigo_tracer/utils.py:103]: it will not work, such like every other configuration that the customer fills mis-correctly... 🤔

Comment (saartochner) [reply src/lumigo_tracer/utils.py:103]: The client needs to implicitly write `timeout_timer_buffer='  '` in order for it to not work...
(empty string and none will work)

Comment (efimk-lu) [reply src/lumigo_tracer/utils.py:103]: I mean, will the tracer behave correctly in case it tries to parse a non float value

Comment (saartochner) [reply src/lumigo_tracer/utils.py:103]: It will raise an exception immediately

Comment (efimk-lu) [reply src/lumigo_tracer/utils.py:103]: That's the problem I was referring to. Bad configuration, just load the default and write to logs print(bla bla) don't put this error message under lumigo debug

Comment (saartochner) [reply src/lumigo_tracer/utils.py]: Updated

**Pull request merged**